.github/workflows/main.yml

```diff
@@ -2,9 +2,46 @@ name: build
 
 on: [push, pull_request]
 
-
+env:
+  PYTEST_ADDOPTS: "--color=yes"
 
 jobs:
+  test:
```

Comment:
Please add an extra line in between the `jobs:` and `test:` lines (makes everything more readable).